certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -52,14 +58,15 @@ def setup_acme_server(acme_server, nodes, proxy=True):
     """
```

Comment:
We should update the documentation for `proxy`.

Reply:
Done.